src/execution/physical_plan/plan_explain.cpp

```diff
@@ -18,10 +18,14 @@ unique_ptr<PhysicalOperator> PhysicalPlanGenerator::CreatePlan(LogicalExplain &o
 	if (context.explain_output_optimized_only) {
```

Comment:
Perhaps change this into a flag "context.explain_output" that is set to either `OPTIMIZED_ONLY, PHYSICAL_ONLY or ALL` (defaults to ALL).